### PR TITLE
Styleguidist frontend

### LINKED ISSUE #169: Build styleguidist documentation for the frontend

### PR DESCRIPTION
**Pull request by robert10290**
closes #169

### CODE REVIEW

File: frontend/src/components/WofButton.vue [hunk 1]
```diff
@@ -1,27 +1,42 @@
...
+     * Size of font and icon.
+     */
     size: {
       type: Number,
       required: false,
       default: null
     },
+    /**
+     * Name of icon.
+     * @example wof-ok-circle:before
```
Comment (karolkozlo) [inline frontend/src/components/WofButton.vue:38]: "wof-ok-circle" is proper name

File: frontend/src/components/WofIcon.vue [hunk 2]
```diff
@@ -5,13 +5,23 @@
...
 <script>
+/**
+ * Component which contain icon.
+ */
 export default {
     name: 'WofIcon',
     props: {
+        /**
+        * Name of icon.
+        * @example wof-ok-circle:before
```
Comment (karolkozlo) [inline frontend/src/components/WofIcon.vue:16]: "wof-ok-circle" is proper name

**Review by karolkozlo — COMMENTED** [review]
It needs two small changes. Everything else is ok.

**Comment by kpagacz** [comment]
Ideally this PR will also include at least a script to build the documentation and at most would also include hosting this documentation on an s3 bucket and an automated workflow to build and update the documentation in that bucket. For a prototype of the architecture, look at what I did with the backend documentation.

**Comment by kpagacz** [comment]
But I guess we don't have much time for anyone to figure this one out. If you feel like you wouldn't be able to do it, then we have abandon the script for now and just merge the docs.

**Review by kpagacz — APPROVED** [review]

**Pull request merged**